src/components/ParticleWrapper.vue

```diff
@@ -1,45 +1,116 @@
 <template>
-      <div id='particle-wrapper'>
-        <!-- <img id="particle-background" src="./assets/graph.png"/> -->
-        <div id="animation">
-          <HeaderAnimation/>    
-        </div>
-        <div id="text">
-          <div class="content">
-            <h1>Find Music – The New Way</h1>
-            <v-btn outline color="#d8ae91" to="/about">More about out-of-tune</v-btn>
-          </div>
-        </div>
-        <div id="particles">
-          <vue-particles
-            color="#dedede"
-            :particleOpacity="0.7"
-            :particlesNumber="80"
-            shapeType="circle"
-            :particleSize="4"
-            linesColor="#dedede"
-            :linesWidth="1"
-            :lineLinked="true"
-            :lineOpacity="0.4"
-            :linesDistance="150"
-            :moveSpeed="3"
-            :hoverEffect="true"
-            hoverMode="grab"
-            :clickEffect="false"
-            clickMode="push"
-          >
-          </vue-particles>  
-        </div>
+  <div id='particle-wrapper'>
+    <!-- <img id="particle-background" src="./assets/graph.png"/> -->
+    <div id="animation">
+      <HeaderAnimation />
+    </div>
+    <div id="text">
+      <div class="content">
+        <!-- <h1>Find Music – The New Way</h1> -->
+        <a href="https://app.out-of-tune.org">
+          <button id="app-button">OOT App</button>
+        </a>
+        <router-link to="/about">
+          <button>About out-of-tune</button>
+        </router-link>
       </div>
+    </div>
+    <vue-particles id="tsparticles" :particlesInit="particlesInit" :particlesLoaded="particlesLoaded" :options="{
+      background: {
+        color: {
+          value: '#1e1e1e'
+        }
+      },
+      fpsLimit: 60,
+      interactivity: {
+        events: {
+          onClick: {
+            enable: true,
+            mode: 'push'
+          },
+          onHover: {
+            enable: true,
+            mode: 'repulse'
+          },
+          resize: true
+        },
+        modes: {
+          bubble: {
+            distance: 150,
+            duration: 2,
+            opacity: 0.8,
+            size: 40
+          },
+          push: {
+            quantity: 4
+          },
+          repulse: {
+            distance: 200,
+            duration: 0.4
+          }
+        }
+      },
+      particles: {
+        color: {
+          value: '#dedede'
+        },
+        links: {
```

Comment:
Have you tried specifying also the width and distance for the links? In the old code you set it explicitly, but I'm not sure what the defaults are.